controllers/neutronapi_controller.go

```diff
@@ -412,12 +412,18 @@ func (r *NeutronAPIReconciler) reconcileInit(
 		ports[metallbcfg.Endpoint] = portCfg
 	}
 
+	routeAnnotations := make(map[string]string)
+	if instance.Spec.RouteAnnotations.Timeout != "" {
+		routeAnnotations["haproxy.router.openshift.io/timeout"] = instance.Spec.RouteAnnotations.Timeout
```

Comment:
Should `haproxy.router.openshift.io/timeout` be a const?

Reply:
I would avoid using string literals generally, yeah. If we envision that we could potentially need more annotations in the future, we could have a list of consts that contain each like:

```
const (
   routeTimeoutAnnotationName string = "haproxy.router.openshift.io/timeout"
)

Then use the const here instead of the string literal.
```